hcl/eval/partial.go

```diff
@@ -1328,6 +1330,16 @@ func trimIgnored(tokens hclwrite.Tokens) hclwrite.Tokens {
 	return tokens[:ignorePos]
 }
 
+func ignorenlc(tokens hclwrite.Tokens) hclwrite.Tokens {
```

Comment:
I have a vague memory that we had some sort of filter function that would accept a list of tokens that it would filter them out...but I'm probably wrong :laughing: